### PR TITLE
rhel: updated instructions

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Changes proposed in this pull request:

- Updated instructions for RHEL 8.6 and derivatives.

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @sandrobonazzola 

This pull request needs review by: @oVirt/ovirt-documentation @almalinux @rocky-linux @navy-linux

### CODE REVIEW

File: source/documentation/common/install/proc-Enabling_the_Red_Hat_Virtualization_Manager_Repositories.adoc [hunk 1]
```diff
@@ -98,7 +98,7 @@ ifdef::ovirt-doc[]
 Ensure the correct repositories are enabled.
 
 For oVirt 4.5:
-If you are going to install on RHEL 8.6 Beta please follow link:/download/install_on_rhel.html[Installing on RHEL] first.
+If you are going to install on RHEL 8.6 or derivatives please follow link:/download/install_on_rhel.html[Installing on RHEL 8.6 or derivatives] first.
```
Comment (apinnick) [inline source/documentation/common/install/proc-Enabling_the_Red_Hat_Virtualization_Manager_Repositories.adoc:101]: What are "derivatives"?

Comment (sandrobonazzola) [reply source/documentation/common/install/proc-Enabling_the_Red_Hat_Virtualization_Manager_Repositories.adoc:101]: Alma Linux, Rocky Linux and so on...

**Review by mwperina — APPROVED** [review]
+1

**Review by apinnick — APPROVED** [review]
LGTM

**Pull request merged**